compose/README.md

```diff
@@ -0,0 +1,102 @@
+# OpenVASD compose definition
+
+This dir contains the compose definition for:
+- docker-compose
+- podman-compose
+
+Requirements:
+- either podman-compose or docker-compose
+- Optional (for tests): openssl, make, curl, jq
+
+It is divided into three definitions:
+1. base.yaml - contains the actual definition of the services starts OpenVASD in http mode
+2. tls.yaml - overrides the settings within base.yaml to start OpenVASD in TLS mode
+3. mtls.yaml - overrides the settings within base.yaml to start OpenVASD in mTLS mode (preferred)
+
+Which means to start openvasd with http:
+
+```
+podman-compose -f base.yaml
```

Comment:
minor detail: 
to actually start the service, users must also add an "up" to make this an actual full working command ;)
(at least with my podman-compose version 1.0.6  on ubuntu - maybe other versions have up as a default that doesn't need to be typed?)